sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -269,6 +276,12 @@ export interface KeyProperties {
     version?: string;
 }
 
+// @public
+export interface KeyReleasePolicy {
+    contentType?: string;
+    data?: Uint8Array;
```

Comment:
It's the data of the KeyReleasePolicy. Seems fairly descriptive. I'd stick as close to the swagger as possible to avoid confusion between API and REST docs. If the name really needs to be changed, I'd recommend working with Hervey to change it in swagger before either GAs. That said, even in swagger this is typical, though usually called just `value`. E.g. most of your array-like response models are actually an object with (often) a single member named `value` that is the array.

Reply:
Makes sense, thanks for the context as well!